tasks/tenant/update-manager-image-in-git/tests/mocks.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
```

Comment:
We should probably have `set -exo pipefail` here as well? Or maybe it doesn't really matter, because it only really makes a difference for the script that actually runs so it doesn't matter if it's done before or after the function definitions? Not sure, but we usually just do it after the shebang, so probably makes sense to have it there.

Reply:
I thought using `set` in the mocks file would just overwrite that in the task (and be less obvious). No? Open to changing it in the script though

Reply:
The way we inject the mock is that it is put at the beginning of the script - on top of the current script. So typically we will have this after applying the mock:
```
#!/usr/bin/env bash
set -eux

[ mock functions ]

#!/usr/bin/env bash
set -eux

[ original script ]
```

So that works just fine. The only issue would be if you set something extra in the mock that is not in the main script.

I checked like 10 different mocks.sh files now and it was in all of them. So I think it's best to stick with it. (But I'm definitely for adding `-o pipeline` to the options as well - this discussion was about whether or not to copy the set command.)

Reply:
added `set -euxo pipefail` to the mocks.sh here